python-sdk/tests_integration/sql/operators/utils.py

```diff
@@ -191,6 +191,8 @@ def _run_task(ti: TaskInstance, session):
     except AirflowSkipException:
         log.info("Task Skipped, continuing")
     except AstroCleanupException:
+        ti.set_state(state=None)
```

Comment:
```suggestion
        ti.set_state(state=None) # Once the exception is raised the cleanup operator is set to FAILED  state, because of which the cleanup operator was not executed at the end.  
```

Reply:
Added comment